### ISSUE TITLE
upload directory without subdirectories

### ISSUE THREAD

**Issue by romain1206** [simulot/immich-go#77]
hello,

i want to upload pictures from directory but i want no upload subdirectories

i use this command

./immich-go -server https://xxxxxxxx -key xxxxxxxxxxxxxxxxxxxxxx upload /photos/2019/

but it upload subdirectory

many thanks

**Comment by simulot** [comment]
This is the default behavior. Anyway, a subtle problem in the command line parsing enable what do you expect:

give a to:
./immich-go -server https://xxxxxxxx/ -key xxxxxxxxxxxxxxxxxxxxxx upload /photos/2019/*.*


I'll implement this correctly soon.

**Comment by romain1206** [comment]
it's not work with the command

-ash: ./immich-go: No such file or directory

**Comment by simulot** [comment]
arf.... the markdown...
```
./immich-go -server https://xxxxxxxx/ -key xxxxxxxxxxxxxxxxxxxxxx upload /photos/2019/*.*
```

**Comment by romain1206** [comment]
hello, it's not working with `*.*`

**Comment by simulot** [comment]
I think I have fixed this issue in the 0.19
